### ISSUE TITLE
Paper's loss function does not match this implementation

### ISSUE THREAD

**Issue by FinAminToastCrunch** [Alphafrey946/PLPPI#4]
According to the paper, there is a 3 part loss function. 

> The loss function for our PLPPI model consists of three
components; the first two terms follow the same loss setup
from DCL-Net and ConvLSTM [12], while the third term
incorporates triplet ranking loss (TRL)

I do not see this implemented. Furthermore, other aspects such as the learning rate and the number of epochs do not match the paper either. Could the repo maintainers provide some guidance?

**Comment by Alphafrey946** [comment]
I will update the repo to reflect the implementation for the TUS-REC challenge.

I reimplemented this for the loss
https://github.com/DIAL-RPI/UCAT/blob/46740b35a978c37c8c50e2e32f5e04c21a06602e/trainer_ucat_PL.py#L172
